api/tests/routes/backoffice_v3/offers_test.py

```diff
@@ -622,3 +623,48 @@ def test_get_edit_form_test(self, legit_user, authenticated_client):
         with assert_num_queries(3):  # session + user + tested_query
             response = authenticated_client.get(form_url)
             assert response.status_code == 200
+
+
+class BatchOfferValidateTest(PostEndpointHelper):
+    endpoint = "backoffice_v3_web.offer.batch_validate_offer"
+    needed_permission = perm_models.Permissions.FRAUD_ACTIONS
+
+    def test_batch_validate_offer(self, legit_user, authenticated_client):
+        offers = offers_factories.OfferFactory.create_batch(10, validation=offers_models.OfferValidationStatus.DRAFT)
```

Comment:
```suggestion
        offers = offers_factories.OfferFactory.create_batch(3, validation=offers_models.OfferValidationStatus.DRAFT)
```
(et pour les suivant aussi)
car sur la volume, on gagnera un peu de temps dans les tests à créer moins d'objets